CHANGELOG.md

```diff
@@ -417,6 +417,7 @@
 - [Initialize Builtins at Native Image build time][3821]
 - [Add the `Self` keyword referring to current type][3844]
 - [Split Atom suggestion entry to Type and Constructor][3835]
+- [Add parent type field to suggestion][3846]
```

Comment:
This should be user-understandable. By user we mean someone who was using Enso 1 time or was never using it.